qa-test-app/src/main/java/com/mapbox/navigation/qa_test_app/view/FeedbackActivity.kt

```diff
@@ -213,19 +217,22 @@ class FeedbackActivity : AppCompatActivity() {
                         .show()
                     return@let
                 }
-                mapboxNavigation.postUserFeedback(
-                    FeedbackEvent.POSITIONING_ISSUE,
-                    "Test feedback",
-                    FeedbackEvent.UI,
-                    null,
-                    emptyArray(),
-                    feedbackMetadata
-                )
-                Toast.makeText(
-                    this,
-                    "Stored feedback has been sent",
-                    Toast.LENGTH_SHORT
-                ).show()
+                binding.mapView.snapshot { bitmap ->
```

Comment:
this's deferred feedback, a screenshot has to be captured at the same time when we calling `provideFeebackMetadataWrapper`